src/Deprecates.jl

```diff
@@ -0,0 +1,4 @@
+using Base: @deprecate
+
+# Now the batch dimension is the last axis!
+@deprecate RunSR(X, y; kw...) EquationSearch(convert(Array, X'), y; kw...)
```

Comment:
`X'` is the adjoint, not the transpose, of a matrix. Subtly different things.
Also I don't think the `convert` should be necessary provided `EquationSearch` takes the argument type to be `AbstractArray`.

Reply:
Ah, good catch! I guess it doesn't matter since we enforce `<:Real` types but still probably good to change since complex numbers would be nice to eventually include.

I convert to Array since I wasn't sure if `X'` or `transpose(X)` is just a new view of the array, or if it actually reshapes it? For example:

```julia
X = randn(Float32, 100, 5);
y = transpose(X)
typeof(y)
```
gives: 
```
LinearAlgebra.Transpose{Float32,Array{Float32,2}}
```
any idea?

Thanks!
Miles

Reply:
Oh, it seems like `copy(y)` is enough to force it to change the shape

Reply:
Okay, pushed!